lessons/module-3/react-1-the-what-and-the-why.md

```diff
@@ -49,19 +48,22 @@ Use the following prompts to help guide your discussion.
 People will define React in many different ways, but at its core, React is:
 
 ```
-A client-side JavaScript framework that allows you to easily and efficiently manipulate the DOM based on application data and how it changes in response to user interaction.
+A client-side JavaScript library that allows you to easily and efficiently manipulate the DOM based on application data and how it changes in response to user interaction.
 ```
 
 Let's break this definition down a little bit.
 
-### What is a framework?
+### Library vs Framework?
 
-The term framework is often used interchangeably with the term 'library', but they're actually a bit different. Libraries are usually a little bit smaller, and generally serve the purpose of providing us with abstractions over complex code that we would otherwise have to write ourselves. (Think jQuery.) Libraries are usually 'syntactic sugar' over something difficult.
+The term 'library' is often used interchangeably with the term 'framework', but they're actually a bit different. Libraries are _usually_ a little bit smaller, and generally serve the purpose of providing us with abstractions over complex code that we would otherwise have to write ourselves. (Think jQuery.) Libraries are sometimes 'syntactic sugar' over something difficult and may target specific functionality.
```

Comment:
Do students still get exposure to jQuery? Maybe we should link to the docs here when we reference it.

Reply:
Great idea - I added a link to the jquery docs.